crates/server/src/zone_handler/catalog.rs

```diff
@@ -1175,6 +1205,10 @@ async fn build_forwarded_response(
     let authorities =
         maybe_strip_dnssec_records(authorities, query.query_type(), lookup_options.dnssec_ok);
 
+    // Strip DNSSEC records from additionals as well.   Is this necessary?
```

Comment:
Is determining the answer to this comments question a blocker for merging or something that can be revisited later?